src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -13,6 +13,9 @@ const formConfig = {
   urlPrefix: '/1990-rjsf/',
   submitUrl: '/v0/education_benefits_claims/1990',
   trackingPrefix: 'edu-',
+  formId: '1990',
+  version: 0,
+  disableSave: true,
```

Comment:
Opting into disabling save in progress while it isn't done makes me nervous. I get that it would make it cleaner later. Maybe we should also put a `__BUILDTYPE__ === 'development'` check where it's used in the common code, too?